### ISSUE TITLE
Hi，When will the code be released? Thank you!

### ISSUE THREAD

**Issue by sev777** [Hunter-DDM/knowledge-neurons#1]
Hi，When will the code be released? Thank you!

**Comment by Hunter-DDM** [comment]
We will release the code before April 20, thank you~

**Comment by sev777** [comment]
Hi, I find in '3_modify_activation.py', the code 
   `                 _, logits = model(input_ids=input_ids, attention_mask=input_mask, token_type_ids=segment_ids, tgt_pos=tgt_pos, tgt_layer=0, imp_pos=kn_bag, imp_op='remove') `
why the tgt_later is always 0, in kn_bag, some neurons is [9,1000] or [10,1001], not always in layer 0.
